backend/alert/alerts.py

```diff
@@ -48,46 +49,134 @@ def __init__(self, template, reg):
                 "auto_resubscribe": reg.auto_resubscribe,
             }
         )
+        self.close_text = None
+        if close_template:
+            t = loader.get_template(close_template)
+            self.close_text = t.render(
+                {
+                    "course": reg.section.full_code,
+                    "brand": "Penn Course Alert",
+                    "auto_resubscribe": reg.auto_resubscribe,
+                }
+            )
         self.registration = reg
 
     @abstractmethod
-    def send_alert(self):
+    def send_alert(self, close_notification=False):
         pass
 
 
 class Email(Alert):
     def __init__(self, reg):
-        super().__init__("alert/email_alert.html", reg)
+        super().__init__(
+            "alert/email_alert.html", reg, close_template="alert/email_close_alert.html"
+        )
 
-    def send_alert(self):
+    def send_alert(self, close_notification=False):
+        """
+        Returns False if notification was not sent intentionally,
+        and None if notification was attempted to be sent but an error occurred.
+        """
         if self.registration.user is not None and self.registration.user.profile.email is not None:
             email = self.registration.user.profile.email
         elif self.registration.email is not None:
             email = self.registration.email
         else:
             return False
+
         try:
+            if close_notification:
+                if not self.close_text:
+                    # This should be unreachable
+                    return None
+                alert_subject = "RE: %s is now open!" % self.registration.section.full_code
```

Comment:
Small thing, but if we want to make sure we're replying to the right thread, should probably pull out `%s is now open!` into its own string so we know it's the same.